manifest_template.yml

```diff
@@ -80,18 +80,36 @@ apigee:
         attributes:
           - name: access
             value: public
-          - name: ratelimit
-            value: {{ ENV.ratelimit | default('5ps') }}
+          - name: ratelimiting
+            value:
+              e-referrals-service-api-{{ VARIANT.service | default(VARIANT.name) }}-{{ ENV.name }}:
+                quota:
+                  enabled: false
+                spikeArrest:
+                  enabled: false
+              app:
+                quota:
+                  enabled: true
+                  interval: 8
+                  limit: {{ ENV.requestLimit | default('288000') }}
+                  timeunit: hour
+                spikeArrest:
+                  enabled: true
+                  ratelimit: {{ ENV.burstLimit | default('720pm') }}
+          - name: requestLimit_binary
+            value: 57600
+          - name: burstLimit_binary
```

Comment:
Non-Binary rate limits are configured app level and binary rate limits are configured in the product level. Is that expected?

Reply:
We decided to leave PerProxy rate limiting for now. Both rate limiting shared flow policies https://apigee.com/platform/nhsd-nonprod/sharedflows/ApplyRateLimiting/overview/41 (non-binary) and binary policies are rate limiting at the app level, by using a reference to <Identifier ref="developer.app.id"/> . 
Although Apigee recommends not using "developer.app.id" https://docs.apigee.com/api-platform/reference/policies/quota-policy#identifierelement , it is what the shared flow uses to rate limit at App level, so for consistency I did the same for our custom binary policies.